src/main/java/org/sagebionetworks/bridge/exporter/dynamo/DynamoHelper.java

```diff
@@ -162,15 +172,31 @@ public StudyInfo getStudyInfo(String studyId) {
             studyIdList.addAll(request.getStudyWhitelist());
         }
 
-        return studyIdList;
+        List<String> studyIdsToQuery = new ArrayList<>();
+
+        for (String studyId : studyIdList) {
+            Item studyIdItem = ddbExportTimeTable.getItem(STUDY_ID, studyId);
+            if (studyIdItem != null) {
+                DateTime lastExportDateTime = new DateTime(studyIdItem.getLong(LAST_EXPORT_DATE_TIME), timeZone);
+                if (!endDateTime.isBefore(lastExportDateTime)) {
+                    studyIdsToQuery.add(studyId);
+                }
+            } else {
+                studyIdsToQuery.add(studyId);
+            }
+
+            TimeUnit.SECONDS.sleep(1);
```

Comment:
It would be better to swallow the InterruptedException and log a message. Otherwise, one failure would fail the entire request and force a redrive.